backend/main.py

```
@@ -22,7 +22,7 @@
 from services.file_service import FileService
 from services.ai_agent import SamuraiAgent
 from services.context_service import context_service
-from services.response_service import handle_long_response, handle_validation_error
+from services.response_service import handle_agent_response, handle_validation_error
```

Ruff (0.12.2)

`services.response_service.handle_validation_error` imported but unused

Remove unused import: `services.response_service.handle_validation_error`

(F401)